e2e/suites/management/utils.go

```diff
@@ -43,7 +43,7 @@ func createSchedulerAndWaitForIt(
 	kubeclient kubernetes.Interface,
 	gruCommand []string) (string, error) {
 	schedulerName := framework.GenerateSchedulerName()
-	createRequest := &maestroApiV1.CreateSchedulerRequest{
+	createRequest := &maestroApiV1.SchedulerRequest{
```

Comment:
Why this was changed? I think this MIGHT break all e2e tests, no?

Reply:
> Why this changed? I think this MIGHT break all e2e tests, no?

good! I had renamed this message to make them more generic, but made rollback and forgot to change this point.